tasty-inspector/src/scala/tasty/inspector/Inspector.scala

```diff
@@ -0,0 +1,25 @@
+package scala.tasty.inspector
+
+import scala.quoted._
+import scala.quoted.runtime.impl.QuotesImpl
+
+import dotty.tools.dotc.Compiler
+import dotty.tools.dotc.Driver
+import dotty.tools.dotc.Run
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Mode
+import dotty.tools.dotc.core.Phases.Phase
+import dotty.tools.dotc.fromtasty._
+import dotty.tools.dotc.util.ClasspathFromClassloader
+import dotty.tools.dotc.CompilationUnit
+import dotty.tools.unsupported
+import dotty.tools.dotc.report
+
+import java.io.File.pathSeparator
+
+trait Inspector:
+
+  /** Inspect all TASTy files using `Quotes` reflect API */
+  def inspect(using Quotes)(tastys: List[(String, quotes.reflect.Tree)]): Unit
```

Comment:
Some documentation of what the string is would be in order, no?

Also, didn't we agree that the method in `Quotes` for obtaining the current compilation unit should be changed to throw an exception when used from Inspector?

Reply:
Added the missing documentation and test and better exception for `reflect.SourceFile.current`.